src/otx/algo/classification/losses/asymmetric_angular_loss_with_ignore.py

```diff
@@ -6,23 +6,77 @@
 from __future__ import annotations
 
 import torch
-from mmpretrain.models.losses.utils import weight_reduce_loss
+
+# TODO(harimkang): Remove the following import after decoupling mmpretrain
 from mmpretrain.registry import MODELS
```

Comment:
Just for a question, where is this registry used?

Reply:
Currently, this PR has only been replaced for the mobilenet-v3-large usage model, so other multi-label classification models still need to be added to the registry.